ansible/roles/azure/files/cloud-init.yml

```diff
@@ -0,0 +1,23 @@
+#cloud-config
```

Comment:
Just to confirm, the `#cloud-config` has to be the first line in the file for Azure to recognize this as a cloud init script rather than a shell script or something else. Is that correct? Not sure on the general convention for cases like this, but I wonder if it should be `#cloud-config` followed by the license, and then the file contents (since the license is usually the first thing in the file)?

Reply:
I took care of this now. It is just the first line that needs to be exactly that string, so the header was moved up.